tests/test_aws_generator.py

```diff
@@ -54,37 +92,51 @@ def setUp(self):
             self.fake.ean(length=13),
             self.fake.ean(length=13),
         )
+        self.TestGenerator = choice(AWS_GENERATORS)
```

Comment:
So, wait are we testing a random aws generator each time we run the test?

Reply:
Yes, for the tests that don't specify which generator to use. Each of the AWSGenerators all behave the same way and use the same interface, so it doesn't matter a whole lot which one we use for some tests.

Reply:
Yeah, I guess my gut feeling was that we should loop through the AWSGenerators instead to confirm that they all work the exact same way. My overall fear is what could happen in the future ya know?

For example, what if someone came in and made a change to code that would cause one of these test to fail, but since the random choice was called on a different generator it made it through pr process. Then we would be trying to debug why our nise tests sometimes randomly fail. 

I wonder if we could use the unittest [subTest](https://github.com/project-koku/koku/blob/b03c6606cc9dbd28560687198722c2a8b3cd7120/koku/api/report/test/tests_queries.py#L52) method to loop through generators.